### DISCUSSION TITLE
Welcome to fire-ui Discussions!

### DISCUSSION THREAD

**Discussion by kimlimjustin** [fire-ui/fire-ui#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Follow our [code of conduct](https://github.com/kimlimjustin/fire-ui/blob/master/CODE_OF_CONDUCT.md)

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by j-blaszczyk** [comment]
Hi, I am pretty new to this project and I am wondering what software do you use for creating one CSS file from all the source code. I can't find any dependencies in the package.json file that could help me. This information could help me in creating some simple testing commands for developing this project.

**Comment by kimlimjustin** [comment]
Hello, basically I just copy and paste them from different files into single FireUI.css file

**Reply by j-blaszczyk** [reply]
Thanks for the information.

**Comment by j-blaszczyk** [comment]
Hey, I was wondering if we could port this project to SCSS. It would be pretty easy because as you may know CSS file is already valid SCSS. It would make it easier to connect all files into one compiled. If everyone agrees on that I could even try to port it myself.

**Reply by kimlimjustin** [reply]
Actually I've planned about that but I got no time, so yeah sure if you want to to help us to port it!

**Comment by LegacyDebaditya** [comment]
I actually visited the website(docs) and saw that the [tooltip docs](https://fire-ui.netlify.app/docs/tooltip) tooltips were actually not in contrast when I switched to dark mode and I think this should be repaired.

**Reply by kimlimjustin** [reply]
Oh, thanks for your notice. I've opened the developer console and I found out that's because a CSS value is mistyped. It supposed to be `var(--black-color) !important` instead of  `var --black-color!important`. This will be fixed soon.  Anyways, thank you very much.

**Reply by kimlimjustin** [reply]
Also, if you found any bugs or if you have any feature request, you are welcomed to open an issue on this repo 😄

**Comment by LegacyDebaditya** [comment]
I would also like to know how did you make theme and where can I get the code for the theme.

**Reply by kimlimjustin** [reply]
It's available on this [file](https://github.com/fire-ui/fire-ui/blob/master/src/feature/Colors_and_Theme/color.scss)
